Dockerfile

```diff
@@ -33,7 +33,15 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master
 # Add the repos containing nginx and galaxy charts
 RUN helm repo add center https://repo.chartcenter.io && \
     helm repo add galaxy https://raw.githubusercontent.com/cloudve/helm-charts/anvil/ && \
+    helm repo add terra-app-setup-charts https://storage.googleapis.com/terra-app-setup-chart && \
```

Comment:
Did you push this manually?

I had created a `terra-app-charts` bucket, wondering if we could use that bucket for both the terra-app and setup charts.

Reply:
yea..I manually pushed to this chart....I wasn't sure if it's supposed to be in the same bucket and was afraid to break the existing chart so I created a separate one

Reply:
I think the same bucket can have multiple charts